hyperdome_client/encryption.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+"""
+Hyperdome
+
+Copyright (C) 2019 Skyelar Craver <scravers@protonmail.com>
+                   and Steven Pitts <makusu2@gmail.com>
```

Comment:
Aww :heart:

Reply:
Would be way to much work to remove at this point :P